### ISSUE TITLE
Megameepo dispellable by many abilities

### ISSUE THREAD

**Issue by Winther12** [ValveSoftware/Dota2-Gameplay#10262]
### Ability name

Megameepo

### Description

So the skill description of megameepo says "cannot be dispelled". But it very much is. As an avid meepo player I've encountered multiple heroes that can dispel the megameepo. 

The abilities that can dispel megameepo all have a common theme, so it should be easy fix. It's abilities that remove hero units from the map. Here's a list of abilities that can dispel megameepo:

Enemy heroes:
Shadow demon: Disruption
Outworld destroyer: Astral imprisonment

Allied heroes:
Pudge: dismember shard
Snapfire: scepter gobble up
Phoenix: scepter supernova

misc:
Weaver: scepter time laps does not work on megameepo
when megameepo is dispelled, meepo units growth from megameepo remain until expired
manta style dispels megameepo
illusion rune dispels megameepo

It's unfun when a 4.2k item becomes completely useless in a match against given heroes. I've also had to completely turn off hero assist as many people don't know that saving abilities like Phoenix ult scepter will actually leave the rest of the meepos vulnerable and kill meepo, even inside the phoenix supernova.  

### Example Match ID (and possibly Timestamp)

7258875755 - 0:00

**Comment by Winther12** [comment]
> #8074

Yeah that's relevant but this is addressing that other heroes can dispel megameepo. Not just items you bought urself.

**Comment by TheZett** [comment]
This has nothing to do with dispelling, MegaMeepo just cancels itself if the main Meepo gets affected by banishment.

All of the listed abilities apply the banishment mechanic, which is why MegaMeepo gets canceled.

**Comment by Winther12** [comment]
> This has nothing to do with dispelling, MegaMeepo just cancels itself if the main Meepo gets affected by banishment.
> 
> All of the listed abilities apply the banishment mechanic, which is why MegaMeepo gets canceled.

**Comment by Winther12** [comment]
> This has nothing to do with dispelling, MegaMeepo just cancels itself if the main Meepo gets affected by banishment.
> 
> All of the listed abilities apply the banishment mechanic, which is why MegaMeepo gets canceled.

I see, but this effectively dispels the megameepo. Banishing the main meepo cancels/removes the megameepo effect, which is what we refer to as dispelling. I don't think its intended that a lvl 1 outworld destroyer can cancel a scepter ability requiring 4.2k gold. There's no other heroes in dota that have this relationship with their scepter ability. Easily canceled/removed by a normal lvl 1 skill. It's like a facelessvoid chronosphere were to be canceled/removed by a single Q ability from an enemy hero,

**Comment by Anutrix** [comment]
Not sure why you are bringing up Chrono. Most channeled ultimates do get interrupted by simple stun. Even CM's Frostbite and Halberd makes Chrono useless. We have a lot of such interactions in Dota. Another example is snapfire insta destroying Phoenix Egg. 
That is why you pick the carry later and that is why you don't pick Meepo against banishment heroes if you plan to buy Aghs.

Thus, the aghs is situational for Meepo just like most other heroes.

Also, not sure when they added this but pressing Alt key when over Megameepo it clearly states, 'MegaMeepo is cancelled if Meepo is banished, hidden or becomes untargetable' so it's an expected behavior.

**Comment by jaredeg** [comment]
This bug is highlighting two bugs with Meepo, the known banishment one.
And an unknown one that happens if you shift Que or cast MegaMeepo while inside Chrono
